grails-app/domain/com/unifina/domain/signalpath/Canvas.groovy

```diff
@@ -67,10 +70,10 @@ class Canvas {
 		id generator: IdGenerator.name // Note: doesn't apply in unit tests
 		json type: 'text'
 		hasExports defaultValue: false
-		example defaultValue: false
 		adhoc defaultValue: false
 		runner index: 'runner_idx'
 		dashboardItems cascade: 'all-delete-orphan'
+		exampleType enumType: "identity", defaultValue: ExampleType.NOT_SET
```

Comment:
Queries to fetch the examples can become slow if there are many rows. Should we add an index for this field?

Reply:
Excellent point!